api/v1alpha1/sparkcluster_types.go

```diff
@@ -79,6 +79,8 @@ type SparkClusterSpec struct {
 
 	GlobalLabels map[string]string `json:"globalLabels,omitempty"`
 
+	EnvoyFilterLabels map[string]string `json:"envoyFilterLabels,omitempty"`
```

Comment:
Maybe add a comment here? this will propagate into the CRD and be availabe for trying to figure out how this field works.

It _seems_ fine to add this here vs embedding. We can group this with a spark-specific istio config in the future and embed that if we want. The front-facing API will probably stay the same.

Reply:
I agree.